actions/envtest/action.yml

```diff
@@ -1,6 +1,6 @@
 name: Setup envtest
 description: A GitHub Action for setting up controller-runtime envtest
-author: Stefan Prodan
+author: Flux authors
```

Comment:
```suggestion
author: Flux project
```

maybe? Not sure what the best option is.

Reply:
project sounds good.